assets/src/edit-story/components/dropDown/index.js

```diff
@@ -0,0 +1,300 @@
+import { debounce } from 'throttle-debounce';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { ReactComponent as DropDownIcon } from '../../icons/dropdown.svg';
+import { useKeyDownEffect } from '../keyboard';
+import useFocusOut from '../../utils/useFocusOut';
+
+const DropDownContainer = styled.div`
+  width: 100px;
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  flex-grow: 1;
+  color: ${({ theme }) => theme.colors.fg.v0};
+  font-family: ${({ theme }) => theme.fonts.body1.font};
+`;
+
+const DropDownSelect = styled.div.attrs({ role: 'Listbox', tabIndex: '0' })`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  flex-grow: 1;
+  background-color: ${({ theme }) => rgba(theme.colors.bg.v0, 0.3)};
+  border-radius: 4px;
+  padding: 2px 0 2px 6px;
+  cursor: pointer;
+
+  ${({ disabled }) =>
+    disabled &&
+    `
+      pointer-events: none;
+      opacity: 0.3;
+    `}
+
+  svg {
+    width: 28px;
+    height: 28px;
+    color: ${({ theme }) => rgba(theme.colors.fg.v1, 0.3)};
+  }
+`;
+
+const DropDownTitle = styled.span`
+  user-select: none;
+  color: ${({ theme }) => theme.colors.fg.v1};
+  font-family: ${({ theme }) => theme.fonts.label.family};
+  font-size: ${({ theme }) => theme.fonts.label.size};
+  line-height: ${({ theme }) => theme.fonts.label.lineHeight};
+  font-weight: ${({ theme }) => theme.fonts.label.weight};
+  letter-spacing: ${({ theme }) => theme.fonts.label.letterSpacing};
+`;
+
+const DropDownListWrapper = styled.div``;
+
+const DropDownList = styled.ul`
+  position: absolute;
+  top: 100%;
+  left: 0;
+  z-index: 1000;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  float: left;
+  min-width: 160px;
+  width: 100%;
+  padding: 5px 0;
+  margin: 2px 0 0;
+  font-size: 14px;
+  text-align: left;
+  list-style: none;
+  background-color: ${({ theme }) => theme.colors.fg.v1};
+  background-clip: padding-box;
+  border-radius: 4px;
+  box-shadow: 0 6px 12px ${({ theme }) => rgba(theme.colors.bg.v0, 0.175)};
+`;
+
+const DropDownItem = styled.li.attrs({ tabIndex: '0', role: 'option' })`
+  letter-spacing: ${({ theme }) => theme.fonts.label.letterSpacing};
+  padding: 16px;
+  margin: 0;
+  font-family: ${({ theme }) => theme.fonts.label.family};
+  font-size: ${({ theme }) => theme.fonts.label.size};
+  line-height: ${({ theme }) => theme.fonts.label.lineHeight};
+  font-weight: ${({ theme }) => theme.fonts.label.weight};
+
+  &:hover {
+    background-color: ${({ theme }) => theme.colors.bg.v11};
+  }
+
+  &:focus {
+    background-color: ${({ theme }) => theme.colors.bg.v11};
+    outline: none;
+  }
+`;
+
+function DropDown({ options, value, onChange, disabled }) {
+  DropDown.wrapperRef = useRef(null);
+  DropDown.selectRef = useRef();
+  DropDown.arrayOfOptionsRefs = [];
+
+  const [openOptions, setOpenOptions] = useState(false);
```

Comment:
Is this a boolean? I though it was an object of options. Please rename to something obvious such as `isOpen` or similar.

Reply:
Fixed.